docs/config/01-configuration-file.md

```diff
@@ -425,7 +425,7 @@ httpsServerOptions: {
 
 **CLI:** `--log-level debug`
 
-**Possible values:**
+**Possible Values:**
```

Comment:
Pull requests usually follow the mantra "one idea -> one PR".  This correction should probably have been a separate PR.

Reply:
Makes sense, sorry about that. Reverted now. Should I squash all my commits?